### DISCUSSION TITLE
guide/drivers/baidu_share

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#260] [Announcements]
# guide/drivers/baidu_share

Driver for BaiduYun share link, supports downloading but not uploading, using an unofficial api. Settings 1. You need to set surl and pwd to browse the directory, and you need t...

https://alist.nn.ci/guide/drivers/baidu_share.html

**Comment by monsoonwind** [comment]
{"errno":-130,"request_id":8780595761758076628,"errtype":"mis_2","server_time":1692327766,"data":{"title":"","expiredType":0,"link_ctime":0,"fileNums":0,"has_more":false,"list":[]}}
出现这个
